components/intro/Intro.i18n.js

```diff
@@ -22,16 +22,23 @@ export default genI18nMessages({
         pyconIntro: 'PyCon Taiwan',
         whatIsPyConAPAC: 'What is PyCon APAC',
         whatIsPyConAPACContent:
-            ' PyCon APAC is the annual gathering of the global Python community. In addition to the wonderful speeches on various aspects of Python, the annual meeting also gathers people who use Python in various fields in Taiwan to exchange experiences and share the latest technological developments.',
+            'PyCon APAC is the annual gathering of the global Python community. ' +
```

Comment:
I'm currently asking for help to revise the translation ([discord link](https://discord.com/channels/752904426057892052/946041922827472897/972428476185448549)).

Reply:
updated
reference: https://discord.com/channels/752904426057892052/946041922827472897/972428476185448549